test/Libraries/RevitIntegrationTests/CoreTests.cs

```diff
@@ -60,7 +60,7 @@ public void CanChangeLacingAndHaveElementsUpdate()
             Assert.IsNotNull(xyzNode);
             
             //test the shortest lacing
-            xyzNode.UpdateValue(new UpdateValueParams("ArgumentLacing", "Auto"));
+            xyzNode.UpdateValue(new UpdateValueParams("ArgumentLacing", "Shortest"));
```

Comment:
Please pull the latest `Revit2017` and resolve this conflict. The latest one should be `"Auto"` :)